<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Led" name="v1_led">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>v1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="Led" name="s11_led">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Led" name="s12_led">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S11</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="cedaTank" name="tank1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>tank1</string>
    </property>
   </widget>
   <widget class="Led" name="s21_led">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Led" name="s22_led">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S22</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="Led" name="b1_led">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S21</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>b1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="cedaTank" name="tank2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>tank2</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="cedaTank" name="tank3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Led" name="v2_led">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>v2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S31</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="Led" name="s31_led">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>221</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>199</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S32</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="Led" name="s32_led">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>tank3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temp_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Led" name="h1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>heater</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="auto_run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatic Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="case_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Test case 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="case_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Test case 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="case_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Test case 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="case_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Test case 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="case_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Test case 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="case_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>set all 0</string>
    </property>
   </widget>
   <zorder>line_2</zorder>
   <zorder>v1_led</zorder>
   <zorder>label</zorder>
   <zorder>s11_led</zorder>
   <zorder>s12_led</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>tank1</zorder>
   <zorder>s21_led</zorder>
   <zorder>s22_led</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>tank2</zorder>
   <zorder>line</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>tank3</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>b1_led</zorder>
   <zorder>v2_led</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>s31_led</zorder>
   <zorder>label_9</zorder>
   <zorder>s32_led</zorder>
   <zorder>label_10</zorder>
   <zorder>temp_1</zorder>
   <zorder>h1</zorder>
   <zorder>label_11</zorder>
   <zorder>auto_run</zorder>
   <zorder>case_1</zorder>
   <zorder>case_2</zorder>
   <zorder>case_3</zorder>
   <zorder>case_4</zorder>
   <zorder>case_5</zorder>
   <zorder>case_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QToolButton</extends>
   <header>led.h</header>
  </customwidget>
  <customwidget>
   <class>cedaTank</class>
   <extends>QToolButton</extends>
   <header>cedatank.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
